<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>qt</class>
    <widget class="QMainWindow" name="qt">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>450</width>
                <height>550</height>
            </rect>
        </property> 
        <property name="styleSheet">
            <string notr="true">
                font-size: 14px;
                color: lightblue;
                background-color: #BFC19B;
            </string>
        </property>
        <property name="cursor">
            <cursorShape>ArrowCursor</cursorShape>
        </property>
        <property name="windowTitle">
            <string>AVR-FLASHER</string>
        </property>
        <widget class="QLabel" name="title_h1">
            <property name="geometry">
                <rect>
                    <x>20</x>
                    <y>10</y>
                    <width>430</width>
                    <height>20</height>
                </rect>
            </property>
            <property name="text">
                <string>AVR-FLASHER - Update your projects</string>
            </property>
            <property name="styleSheet">
                <string notr="true">
                    text-align: center;
                    font-size: 16px;
                    color: #222587;
                </string>
            </property>
        </widget>
        <widget class="QLabel" name="bauds_label">
            <property name="geometry">
                <rect>
                    <x>40</x>
                    <y>40</y>
                    <width>50</width>
                    <height>30</height>
                </rect>
            </property>
            <property name="text">
                <string>Bauds:</string>
            </property>
            <property name="styleSheet">
                <string notr="true">
                    text-align: center;
                    font-size: 16px;
                    color: black;
                </string>
            </property>
        </widget>
        <widget class="QPushButton" name="set_19200_bauds">
            <property name="geometry">
                <rect>
                    <x>100</x>
                    <y>45</y>
                    <width>60</width>
                    <height>20</height>
                </rect>
            </property>
            <property name="styleSheet">
                <string notr="true">
                    font-size: 12px;
                    color: black;
                    background-color: lightgrey;
                </string>
            </property>
            <property name="text">
                <string>19200</string>
            </property>
        </widget>
        <widget class="QPushButton" name="set_115200_bauds">
            <property name="geometry">
                <rect>
                    <x>180</x>
                    <y>45</y>
                    <width>60</width>
                    <height>20</height>
                </rect>
            </property>
            <property name="styleSheet">
                <string notr="true">
                    font-size: 12px;
                    color: black;
                    background-color: lightgrey;
                </string>
            </property>
            <property name="text">
                <string>115200</string>
            </property>
        </widget>
        <widget class="QPushButton" name="set_1000000_bauds">
            <property name="geometry">
                <rect>
                    <x>260</x>
                    <y>45</y>
                    <width>60</width>
                    <height>20</height>
                </rect>
            </property>
            <property name="styleSheet">
                <string notr="true">
                    font-size: 12px;
                    color: black;
                    background-color: #70C976;
                </string>
            </property>
            <property name="text">
                <string>1000000</string>
            </property>
        </widget>
        <widget class="QPushButton" name="research_port_button">
            <property name="geometry">
                <rect>
                    <x>360</x>
                    <y>40</y>
                    <width>70</width>
                    <height>30</height>
                </rect>
            </property>
            <property name="styleSheet">
                <string notr="true">
                    font-size: 14px;
                    color: black;
                    background-color: lightgrey;
                </string>
            </property>
            <property name="text">
                <string>CONNECT</string>
            </property>
        </widget>
        <widget class="QLabel" name="loadfile_label">
            <property name="geometry">
                <rect>
                    <x>60</x>
                    <y>80</y>
                    <width>350</width>
                    <height>30</height>
                </rect>
            </property>
            <property name="text">
                <string>Cliquez ici pour charger votre programme.hex</string>
            </property>
            <property name="styleSheet">
                <string notr="true">
                    text-align: center;
                    font-size: 14px;
                    color: #151517;
                </string>
            </property>
        </widget>
        <widget class="QPushButton" name="button_open_file">
            <property name="geometry">
                <rect>
                    <x>60</x>
                    <y>105</y>
                    <width>350</width>
                    <height>40</height>
                </rect>
            </property>
            <property name="styleSheet">
                <string notr="true">
                    font-size: 16px;
                    color: black;
                    background-color: lightgrey;
                </string>
            </property>
            <property name="text">
                <string>Chargez votre programme.hex</string>
            </property>
        </widget>
        <widget class="QLabel" name="loading_filename">
            <property name="geometry">
                <rect>
                    <x>80</x>
                    <y>150</y>
                    <width>330</width>
                    <height>30</height>
                </rect>
            </property>
            <property name="text">
                <string>Programme ???</string>
            </property>
            <property name="styleSheet">
                <string notr="true">
                    text-align: center;
                    font-size: 18px;
                    padding-left: 15px;
                    color: black;
                    background-color: grey;
                </string>
            </property>
        </widget>
       <widget class="QLabel" name="loadeeprom_label">
            <property name="geometry">
                <rect>
                    <x>60</x>
                    <y>190</y>
                    <width>350</width>
                    <height>30</height>
                </rect>
            </property>
            <property name="text">
                <string>Cliquez ici pour charger votre EEPROM.hex</string>
            </property>
            <property name="styleSheet">
                <string notr="true">
                    text-align: center;
                    font-size: 14px;
                    color: #151517;
                </string>
            </property>
        </widget>
        <widget class="QPushButton" name="button_open_eeprom">
            <property name="geometry">
                <rect>
                    <x>60</x>
                    <y>215</y>
                    <width>350</width>
                    <height>40</height>
                </rect>
            </property>
            <property name="styleSheet">
                <string notr="true">
                    font-size: 16px;
                    color: black;
                    background-color: lightgrey;
                </string>
            </property>
            <property name="text">
                <string>Chargez votre EEPROM.hex</string>
            </property>
        </widget>
        <widget class="QLabel" name="loading_eeprom">
            <property name="geometry">
                <rect>
                    <x>80</x>
                    <y>260</y>
                    <width>330</width>
                    <height>30</height>
                </rect>
            </property>
            <property name="text">
                <string>EEPROM ???</string>
            </property>
            <property name="styleSheet">
                <string notr="true">
                    text-align: center;
                    font-size: 18px;
                    padding-left: 15px;
                    color: black;
                    background-color: grey;
                </string>
            </property>
        </widget>
        <widget class="QLabel" name="save_eeprom_label">
            <property name="geometry">
                <rect>
                    <x>60</x>
                    <y>320</y>
                    <width>350</width>
                    <height>30</height>
                </rect>
            </property>
            <property name="text">
                <string>1 - Pr√©levez l'EEPROM actuelle</string>
            </property>
            <property name="styleSheet">
                <string notr="true">
                    text-align: center;
                    font-size: 16px;
                    color: #151517;
                </string>
            </property>
        </widget>
        <widget class="QPushButton" name="save_eeprom">
            <property name="geometry">
                <rect>
                    <x>100</x>
                    <y>350</y>
                    <width>270</width>
                    <height>30</height>
                </rect>
            </property>
            <property name="styleSheet">
                <string notr="true">
                    font-size: 16px;
                    color: black;
                    background-color: lightyellow;
                </string>
            </property>
            <property name="text">
                <string>Attente connexion !</string>
            </property>
        </widget>
        <widget class="QLabel" name="flash_label">
            <property name="geometry">
                <rect>
                    <x>60</x>
                    <y>380</y>
                    <width>350</width>
                    <height>30</height>
                </rect>
            </property>
            <property name="text">
                <string>2 - Mettez √† jour votre projet</string>
            </property>
            <property name="styleSheet">
                <string notr="true">
                    text-align: center;
                    font-size: 16px;
                    color: #151517;
                </string>
            </property>
        </widget>
        <widget class="QPushButton" name="flash">
            <property name="geometry">
                <rect>
                    <x>100</x>
                    <y>410</y>
                    <width>270</width>
                    <height>30</height>
                </rect>
            </property>
            <property name="styleSheet">
                <string notr="true">
                    font-size: 12px;
                    color: black;
                    background-color: lightyellow;
                </string>
            </property>
            <property name="text">
                <string>Attente fichier .hex !</string>
            </property>
        </widget>
        <widget class="QLabel" name="eeprom_injection_label">
            <property name="geometry">
                <rect>
                    <x>60</x>
                    <y>440</y>
                    <width>350</width>
                    <height>30</height>
                </rect>
            </property>
            <property name="text">
                <string>3 - Inject√© votre configuration</string>
            </property>
            <property name="styleSheet">
                <string notr="true">
                    text-align: center;
                    font-size: 16px;
                    color: #151517;
                </string>
            </property>
        </widget>
        <widget class="QPushButton" name="eeprom_injection">
            <property name="geometry">
                <rect>
                    <x>100</x>
                    <y>470</y>
                    <width>270</width>
                    <height>30</height>
                </rect>
            </property>
            <property name="styleSheet">
                <string notr="true">
                    font-size: 16px;
                    color: black;
                    background-color: lightyellow;
                </string>
            </property>
            <property name="text">
                <string>Attente fichier .hex !</string>
            </property>
        </widget>
        <widget class="QStatusBar" name="statusbar"/>
        </widget>
    <resources/>
    <connections/>
</ui>
